keras-sprint/deeplabv3_plus.ipynb

```diff
@@ -0,0 +1,4573 @@
+{
```

Comment:
<p>Maybe mention at begining of this section that readers can skip it as this is particular for this model</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/community-events/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>